src/BloomBrowserUI/bookEdit/sourceBubbles/BloomSourceBubbles.ts

```diff
@@ -133,7 +133,7 @@ export default class BloomSourceBubbles {
         items = BloomSourceBubbles.SmartOrderSourceTabs(items, newIso);
 
         var shellEditingMode = false;
-        var list = $this.find('ul');
+        var list = $this.find('nav').find('ul');
```

Comment:
No preference on my part. I was being consistent with the fix John Thompson already made in this file. I just found more places the same fix was needed.